docker-compose.yml

```diff
@@ -370,9 +385,22 @@ services:
       - "8126"
     ports:
       - "127.0.0.1:${DD_METRIC_AGENT_PORT}:8125/udp"
-      - "127.0.0.1:${DD_TRACE_AGENT_PORT}:8126"
+      - "127.0.0.1:8126:8126"
     volumes:
       - ddagent_var_run:/var/run/datadog
+
+  testagent:
+    image: ghcr.io/datadog/dd-apm-test-agent/ddapm-test-agent:latest
+    ports:
+        - "127.0.0.1:9126:9126"
```

Comment:
I suggest creating a new `DD_TEST_AGENT_PORT` variable to configure the host port.

Reply:
```suggestion
        - "127.0.0.1:${DD_TEST_AGENT_PORT}:9126"
```

Reply:
added this